package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
```

Comment:
Delete this file and `yarn.lock` file. We don't have any js code in main folder.

Reply:
I think this is not resolved yet